src/images/mobile-icons/circle_fill.svg

```diff
@@ -0,0 +1,4 @@
+<svg viewBox="0 0 512 512" xmlns="http://www.w3.org/2000/svg">
+  <title>circle fill</title>
+  <ellipse cx="256" cy="256.004" rx="157.538" ry="157.541" fill="black"/>
```

Comment:
I think we can remove fill from all icons so if they are moved to the web icons no one will forget about it.